src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -86,6 +86,32 @@ private static string GetRequestParameterName(RequestParameter requestParameter)
             return language.SerializedName ?? language.Name;
         }
 
+        public CSharpType? GetReturnType(InputOperation operation, string defaultNamespace)
+        {
+            var bodyType = operation.Responses.FirstOrDefault()?.BodyType;
+            if (bodyType != null && bodyType is InputModelType)
+            {
+                return new CSharpType(new ModelTypeProvider((bodyType as InputModelType)!, _typeFactory, defaultNamespace, null));
+            }
+
+            return null;
+        }
+
+        public Parameter? GetBodyParameter(InputOperation operation, string defaultNamespace)
+        {
+            var requestParameters = operation.Parameters
+                .Where(rp => !IsIgnoredHeaderParameter(rp));
+
+            var bodyParameters = requestParameters.Where(parameter => parameter.Location == RequestLocation.Body);
+            var requiredParameter = bodyParameters.Where(parameter => parameter.IsRequired).FirstOrDefault();
+
+            var bodyParameter = requiredParameter ?? bodyParameters.FirstOrDefault();
```

Comment:
When there is no body parameter? it will return default. What is the default value? null?
if so, need to check in following code